src/rules/convert_require/rojo_sourcemap.rs

```diff
@@ -128,8 +164,25 @@ impl RojoSourcemap {
         let from_file = from_file.as_ref();
         let target_file = target_file.as_ref();
 
+        let binding = from_file.to_path_buf().join(target_file);
+
+        let normalized = normalize_path(binding.as_path());
+
         let from_node = self.find_node(from_file)?;
-        let target_node = self.find_node(target_file)?;
+        let target_node = self.find_node(if from_file.is_absolute() {
+            if from_file.is_absolute() {
```

Comment:
I think here we may want to use `has_root()`? It's a bit more flexible and it makes it possible to write paths that starts with `"\"` without using the disk prefix